### PR TITLE
Person details and pagination pages

### PR DESCRIPTION
**Pull request by lkkolosowski**

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -3,13 +3,15 @@ import createSagaMiddleware from "redux-saga";
 import rootSaga from "./rootSaga";
 import moviesListReducer from "./features/moviesListSlice";
 import movieDetailsReducer from "./features/movieDetailsSlice";
+import personDetailsReducer from "./features/personSlice";
```
Comment (Albrecht-Albi) [inline src/store.js:6]: You've chosen a good folder for the file.

File: src/features/personSlice.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    },
+});
+export const {
+    fetchDataSuccess,
+    getDetailsForPerson,
+    fetchDataError,
+} = personSlice.actions;
+
+export const selectPersonState = (state) => state.personDetails;
+export const selectPersonId = (state) => selectPersonState(state).personId;
```
Comment (Albrecht-Albi) [inline src/features/personSlice.js:37]: You've created a great person slice, bravo, well done !!!!

File: src/common/Pagination/index.js [hunk 3]
```diff
@@ -7,31 +8,40 @@ import {
...
-      <Text>of</Text>
-      <PageText>500</PageText>
-      <Button>
+      <Wrapper>
+        <Text>Page</Text>
+        <PageText>{pageNumber}</PageText>
+        <Text>of</Text>
+        <PageText>{totalPages}</PageText>
+      </Wrapper>
+      <Button disabled={pageNumber === totalPages} onClick={toNextPage}>
```
Comment (PawelDzi) [inline src/common/Pagination/index.js:36]: nice, will be pagination

File: src/features/PersonDetails/index.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+              </List>
+            </>
+          )}
+        </>
+      }
+    />
+  );
+};
+
+export default PersonDetails;
```
Comment (PawelDzi) [inline src/features/PersonDetails/index.js:95]: Nice to see people added

File: src/features/getDataFromAPI.js [hunk 5]
```diff
@@ -3,9 +3,9 @@ import axios from "axios";
 const URL = "https://api.themoviedb.org/3";
 const API_KEY = "1a89c6b78383e65e62937e5f4ffe7915";
 
-export const getPopularMovies = async () => {
+export const getPopularMovies = async (page) => {
```
Comment (Albrecht-Albi) [inline src/features/getDataFromAPI.js:6]: Good idea to put page

**Review by PawelDzi — APPROVED** [review]

File: src/features/PersonDetails/styled.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  padding: 0;
+  list-style-type: none;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
+    gap: 16px;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+    grid-template-columns: 1fr;
```
Comment (Albrecht-Albi) [inline src/features/PersonDetails/styled.js:17]: You did a very good job of adding the low resolution mapping

File: src/features/PersonDetails/index.js (hunk 4 above)
Comment (lkkolosowski) [reply src/features/PersonDetails/index.js:95]: Im happy too

File: src/common/Pagination/index.js [hunk 7]
```diff
@@ -7,31 +8,40 @@ import {
...
-      <PageText>1</PageText>
-      <Text>of</Text>
-      <PageText>500</PageText>
-      <Button>
+      <Wrapper>
+        <Text>Page</Text>
+        <PageText>{pageNumber}</PageText>
+        <Text>of</Text>
+        <PageText>{totalPages}</PageText>
+      </Wrapper>
```
Comment (Albrecht-Albi) [inline src/common/Pagination/index.js:35]: Extra we have active pagination, good job man

**Review by Albrecht-Albi — APPROVED** [review]

**Pull request merged**